client/src/components/Applications/AppForm/AppFormUtils.js

```diff
@@ -60,11 +71,7 @@ export const createMaxRunTimeRegex = (maxRunTime) => {
  */
 export const getNodeCountValidation = (queue, app) => {
   // all queues have a min node count of 1 except for the normal queue on Frontera which has a min node of 3
```

Comment:
>   // all queues have a min node count of 1 except for the normal queue on Frontera which has a min node of 3

Could remove this comment now that it's within `getMinNodeCount`

Reply:
Yes. good catch